conda_recipe/environment.yml

```diff
@@ -25,6 +25,7 @@ dependencies:
  - libxml2>=2.9.10
  - llvmdev>=8
  - make
+ - ninja
```

Comment:
I worry about including this here because it will mean that cmake may choose ninja automatically for people who don't specify. And that would be confusing since the default has been make and I don't see a real gain in changing that.

Reply:
@zhipeng-jia Can you address this?

Reply:
This is fine as long as `make` is preferred. Thanks for checking.